config.xml

```diff
@@ -96,4 +95,10 @@
     </plugin>
     <plugin name="cordova-plugin-device-authentication" spec="~0.0.1" />
     <plugin name="cordova-plugin-globalization" spec="^1.11.0" />
+    <plugin name="cordova-plugin-file" spec="^6.0.1" />
+    <plugin name="cordova-plugin-media" spec="^5.0.2" />
+    <allow-navigation href="http://172.27.237.231:8100" />
+    <allow-navigation href="http://192.168.14.205:8100" />
```

Comment:
these ips match your local network, these might not to work with on other machines/networks